server/src/test/java/org/spine3/server/storage/RecordStorageShould.java

```diff
@@ -407,5 +484,13 @@ public Timestamp getNow() {
         public Project getCounterState() {
             return getState();
         }
+
+        public int getInjectableState() {
+            return injectableState;
+        }
+
+        private void injectInjectableState(int state) {
```

Comment:
This isn't good. 

I don't really understand why you would need some `int` field anyway. Couldn't you reuse the `Project`-typed state and use `updateState()` if you really need to do that?